packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -3453,3 +3453,11 @@ export const PARTITION_SAML_SIGN_ON_URL: Record<Partition, string> = {
   [Partition.UsIso]: 'https://signin.c2shome.ic.gov/saml',
   [Partition.UsIsoB]: 'https://signin.sc2shome.sgov.gov/saml',
 };
+
+export const LATEST_NODE_RUNTIME_MAP: Record<Partition, string> = {
+  [Partition.Default]: 'nodejs20.x',
```

Comment:
Default cannot be node20, it must be node18

Reply:
This is not named well but it is not the default. `Partition.Default` is for commercial regions. Our true default is set as node18 in the `regionalMap` API.

Reply:
`Partition.Default` refers to the default partition not the default runtime.